### PR TITLE
[Feature:Forum] Make post order clear for dropdown

### PR DESCRIPTION
**Pull request by 04vmatsibekker**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
The dropdown was a bit confusing and made users believe it was supposed to order the threads not just the posts.
image

### What is the new behavior?
The post ordering is specified. Also, the drop-down is aligned to the right.
image

### CODE REVIEW

**Review by Youssef-Chip — APPROVED** [review]
The new changes specify clearly now that the users are sorting the posts and not the threads.
It looks good.

File: site/app/templates/forum/ForumBar.twig [hunk 1]
```diff
@@ -86,13 +86,13 @@
...
 						</a>
 						<div class="dropdown-divider"></div>
-						<a class="key_to_click dropdown-item"  id="tree" title="Sort posts by reply hierarchy" onclick="changeDisplayOptions('tree', {{ current_thread }})" href="#">Hierarchical</a>
-						<a href="#" id="time" class="key_to_click dropdown-item" tabindex="0" title="Sort posts by ascending chronological order" onclick="changeDisplayOptions('time', {{ current_thread }})">Chronological  <i class="fas fa-angle-up"></i> </a>
-						<a href="#" id="reverse-time" class="key_to_click dropdown-item" tabindex="0" title="Sort posts by descending chronological order" onclick="changeDisplayOptions('reverse-time', {{ current_thread }})">Chronological <i class="fas fa-angle-down"></i> </a>
+						<a class="key_to_click dropdown-item"  id="tree" title="Sort posts by reply hierarchy" onclick="changeDisplayOptions('tree', {{ current_thread }})" href="#">Hierarchical Post Order</a>
+						<a href="#" id="time" class="key_to_click dropdown-item" tabindex="0" title="Sort posts by ascending chronological order" onclick="changeDisplayOptions('time', {{ current_thread }})">Chronological Post Order  <i class="fas fa-angle-up"></i> </a>
```
Comment (bmcutler) [inline site/app/templates/forum/ForumBar.twig:90]: you've got extra spaces after some of these labels, I assume that's not important/intentional?

**Review by bmcutler — COMMENTED** [review]
new screenshot looks good!

File: site/app/templates/forum/ForumBar.twig (hunk 1 above)
Comment (bmcutler) [reply site/app/templates/forum/ForumBar.twig:90]: ok, thanks, didn't catch that

**Pull request merged**